src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/__tests__/ChartBoundaryLevelsConfiguration.test.tsx

```diff
@@ -144,62 +150,79 @@ describe('ChartBoundaryLevelsConfiguration', () => {
     expect(row1Cells[0]).toHaveTextContent(
       'Number of authorised absence sessions (Ethnicity Major Chinese, State-funded primary, All locations, 2014/15)',
     );
-    expect(row1Cells[1]).toHaveTextContent('Use default');
+    expect(within(row1Cells[1]).getByRole('combobox')).toHaveValue('');
+
+    const row2Cells = within(rows[2]).getAllByRole('cell');
+    expect(row2Cells[0]).toHaveTextContent(
+      'Number of authorised absence sessions (Ethnicity Major Chinese, State-funded primary, All locations, 2015/16)',
+    );
+    expect(within(row2Cells[1]).getByRole('combobox')).toHaveValue('');
+
+    rows.forEach((row, rowIndex) => {
+      if (rowIndex === 0) return;
+      const cells = within(row).getAllByRole('cell');
+      const boundaryLevelOptions = within(cells[1]).getAllByRole('option');
+      boundaryLevelOptions.forEach((option, optionIndex) => {
+        if (optionIndex === 0) {
+          expect(option).toHaveTextContent('Use default');
+          expect(option).toHaveValue('');
+        } else {
+          const { id, label } = testMeta.boundaryLevels[optionIndex - 1];
+          expect(option).toHaveTextContent(label);
+          expect(option).toHaveValue(String(id));
+        }
+      });
+    });
   });
 
-  test('renders correctly with initial values', () => {
+  test('renders data sets with initial boundary levels', () => {
     render(
       <ChartBoundaryLevelsConfiguration
         map={{
           dataSetConfigs: [
             {
               dataGrouping: defaultDataGrouping,
               dataSet: testDataSets[0],
-              boundaryLevel: 3,
+              boundaryLevel: testMeta.boundaryLevels[0].id,
             },
             {
               dataGrouping: defaultDataGrouping,
               dataSet: testDataSets[1],
-              boundaryLevel: 3,
+              boundaryLevel: testMeta.boundaryLevels[1].id,
             },
           ],
         }}
         meta={testMeta}
         options={{
           ...testDefaultChartOptions,
-          boundaryLevel: 2,
+          boundaryLevel: testMeta.boundaryLevels[2].id,
         }}
         onChange={noop}
         onSubmit={noop}
         hasDataSetBoundaryLevels
       />,
     );
 
-    expect(screen.getByLabelText('Default boundary level')).toHaveValue('2');
+    expect(screen.getByLabelText('Default boundary level')).toHaveValue(
+      String(testMeta.boundaryLevels[2].id),
```

Comment:
Nitpick, but you could use the `toString` method to convert this to a string. It personally reads a bit nicer, but it doesn't massively make a difference in this case.

Similar could be done below

Reply:
will bare in mind for future!